Code/Tools/Libs/GuiFoundation/PropertyGrid/Implementation/PropertyWidget.cpp

```diff
@@ -552,9 +552,29 @@ void ezQtPropertyEditorIntSpinboxWidget::OnInit()
     {
       case 1:
       {
+        const ezInt32 iMinValue = pClamp->GetMinValue().ConvertTo<ezInt32>();
+        const ezInt32 iMaxValue = pClamp->GetMaxValue().ConvertTo<ezInt32>();
+
         ezQtScopedBlockSignals bs(m_pWidget[0]);
         m_pWidget[0]->setMinimum(pClamp->GetMinValue());
         m_pWidget[0]->setMaximum(pClamp->GetMaxValue());
+
+        if ((iMaxValue - iMinValue) < 256)
```

Comment:
I'd change this to
```
if (pClamp->GetMinValue().IsValid() && pClamp->GetMaxValue().IsValid())
```
`ConvertTo<ezInt32>()` will return 0 if the value can't be converted.

Reply:
Fixed, thanks!